### PR TITLE
Proposed fix for #986.

### PR DESCRIPTION
**Pull request by rgobbel**
No metaprogramming required now. Fix for #986. No changes were required other than deleting these few lines.

### CODE REVIEW

File: src/stdio.jl [hunk 1]
```diff
@@ -23,16 +23,6 @@ Base.displaysize(io::IJuliaStdio) = displaysize(io.io)
 Base.unwrapcontext(io::IJuliaStdio) = Base.unwrapcontext(io.io)
 Base.setup_stdio(io::IJuliaStdio, readable::Bool) = Base.setup_stdio(io.io.io, readable)
 
-for s in ("stdout", "stderr", "stdin")
```
Comment (fredrikekre) [inline src/stdio.jl:26]: You should put this in a conditional to still support older Julia version:
```
if VERSION < v"1.7.0-DEV.254"
    # ...
end
```

Comment (rgobbel) [reply src/stdio.jl:26]: Ah, of course. Done, testing now...

File: src/stdio.jl [hunk 2]
```diff
@@ -23,13 +23,15 @@ Base.displaysize(io::IJuliaStdio) = displaysize(io.io)
...
+if VERSION < v"1.7.0-DEV.254"
+    for s in ("stdout", "stderr", "stdin")
+        f = Symbol("redirect_", s)
+        sq = QuoteNode(Symbol(s))
+        @eval function Base.$f(io::IJuliaStdio)
+            io[:jupyter_stream] != $s && throw(ArgumentError(string("expecting ", $s, " stream")))
+            Core.eval(Base, Expr(:(=), $sq, io))
+            return io
+        end
     end
```
Comment (stevengj) [inline src/stdio.jl:35]: Shouldn't there be an `else` clause to do something new in Julia 1.7?

Comment (rgobbel) [reply src/stdio.jl:35]: There's no need. This code added functionality that is now part of the standard versions of these functions.

**Pull request merged**